srtcore/srt_compat.c

```diff
@@ -17,7 +17,7 @@ written by
 // Prevents from misconfiguration through preprocessor.
 
 #include "platform_sys.h"
-
+#include <srt.h>
```

Comment:
Why is `srt.h` needed in this file?

Reply:
SRT_API seems to be needed in srt_compat.h. I don't know why I added it in the c file instead, should I move it ?

Reply:
Right, but then I doubt that `SysStrError` really requires this attribute. This is an internally used function, not a part of the SRT API. I believe it should be able to go without it, but I'd check this on all Windows builds to be sure first.

Reply:
All AppVeyor build for windows went fine